src/main.cpp

```diff
@@ -1,9 +1,117 @@
 #include "cloud4rpi.h"
 
-Cloud4RPi::Cloud4RPi(const char *deviceToken, const char* server) {
+Cloud4RPi::Cloud4RPi(const char *deviceToken, Client *client, const char* server) {
     _deviceToken = deviceToken;
     _server = server;
+    _mqttClient = new PubSubClient(*client);
+    _mqttClient->setServer(_server, 1883);
+    _jsonBufferSize = JSON_BUFFER_SIZE;
 }
+
+void Cloud4RPi::declareVariables(Cloud4RPiVar variablesArr[], int variableCount) {
+    _variables = variablesArr;
+    _variableCount = variableCount;
+}
+
+String Cloud4RPi::getVariable(const char *varName) {
+    for(int n = 0; n < _variableCount; n++) {
+        if(strcmp(_variables[n].name, varName) == 0) {
+            return _variables[n].value;
+        }
+    }
```

Comment:
maybe we can avoid redundant brackets? Personally I don't like them at all and tend to ommit every single bracket if possible.